src/main/java/com/google/plantasy/utils/UserUtils.java

```diff
@@ -163,20 +165,22 @@ public static boolean addBlobKey(String blobKey, Entity userEntity, DatastoreSer
         return false;
     }
     
-    userEntity.setProperty("blobKey", blobKey);
-    datastore.put(userEntity);
- 
+    User user = new User(userEntity);
+    user.setBlobKey(blobKey);
+
     return true; 
   }
  
   /**
     adds a specified number of points to the user's points
   */
   public static void addPoints(Entity userEntity, int numPoints, DatastoreService datastore){
```

Comment:
AddPoints() should be changed to take in a User and not a UserEntity (this means modifying line 214 in this file to pass user and not userEntity.)

Reply:
Okay, that's done and changed a few places where it's called to match that